modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -103,6 +103,8 @@ An egress firewall has the following limitations:
 
 ifdef::ovn[]
 * A maximum of one {kind} object with a maximum of 8,000 rules can be defined per project.
+
+* If you are using the OVN-Kubernetes network plugin with shared gateway mode in Red Hat OpenShift Networking, when ingress traffic arrives at a pod affected by the egress firewall, the return ingress reply is affected by the egress firewall rules. If the egress firewall rules drop the ingress reply destination IP, the traffic is dropped. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2047299[*BZ#2047299*])
```

Comment:
Okay... so my comment somehow didn't get submitted :crying_cat_face: , so let me try again...

* plugin -> plug-in
* We don't normally put BZ links in the main doc text. Usually if it's a known issue it would go into the release notes
* The first sentence is pretty long and hard to follow. Took a stab at dropping some/rephrasing, in case it gets the same meaning across

```suggestion
* If you are using the OVN-Kubernetes network plug-in with shared gateway mode in Red Hat OpenShift Networking, return ingress replies are affected by egress firewall rules. If the egress firewall rules drop the ingress reply destination IP, the traffic is dropped.
```